homeworks/Yaroslav.Belinskyi_YaroslavBelinskyi/homework_7/script.js

```diff
@@ -0,0 +1,16 @@
+function randomEndValue() {
+    return Math.floor(Math.random() * 51 + 50);
+}
+function fibonacci() {
+    const fibonacciArray = [0, 1];
+    const endOfArray = randomEndValue();
+    for (let i = 0; ;) {
+        i = fibonacciArray[fibonacciArray.length - 1] + fibonacciArray[fibonacciArray.length - 2];
+        if (i < endOfArray) {
+            fibonacciArray.push(i);
+        } else { break; }
```

Comment:
if you have to do thing like this a loop body, it means something may be designed better